static/js/brand-store/Snaps/Snaps.js

```diff
@@ -155,7 +155,7 @@ function Snaps() {
       });
   };
 
-  const isAdmin = () => currentMember.roles.includes("admin");
+  const isAdmin = () => currentMember?.roles.includes("admin");
```

Comment:
If `currentMember.roles` is undefined it'll return `undefined` whereas if it hits the `includes()` call it'll return a `boolean`.